package.json

```diff
@@ -15,14 +15,14 @@
   },
   "scripts": {
     "postinstall": "patch-package",
-    "test": "npm run task test",
-    "build": "npm run task build",
-    "task": "npm run prep && ts-node -T ./src/tasks/tasks.ts",
-    "cli": "npm run prep && ts-node -T ./src/lib/cli.ts",
-    "prep": "npm install --no-audit && ts-node -T src/lib/checkEngine.ts",
-    "demo-start": "npm run demo-prep && ts-node -T ./src/demo/run-script.ts start",
-    "demo-build": "npm run demo-prep && ts-node -T ./src/demo/run-script.ts build",
-    "demo-prep": "npm run task -- build --no-test && npm i --prefix src/demo"
+    "test": "yarn task test",
+    "build": "yarn task build",
+    "task": "yarn prep && ts-node -T ./src/tasks/tasks.ts",
+    "cli": "yarn prep && ts-node -T ./src/lib/cli.ts",
+    "prep": "yarn install --no-audit && ts-node -T src/lib/checkEngine.ts",
+    "demo-start": "yarn demo-prep && ts-node -T ./src/demo/run-script.ts start",
+    "demo-build": "yarn demo-prep && ts-node -T ./src/demo/run-script.ts build",
+    "demo-prep": "yarn task -- build --no-test && yarn install --prefix src/demo"
```

Comment:
`--prefix` doesn't work for yarn as far as I know.
Should probably be `cwd`: https://classic.yarnpkg.com/en/docs/cli/#toc-cwd